Packaging/OpenDingux/build.sh

```diff
@@ -60,7 +60,12 @@ make_buildroot() {
 	cp buildroot_${TARGET}_defconfig "$BUILDROOT/configs/${TARGET}_devilutionx_defconfig"
 	cd "$BUILDROOT"
 	make ${TARGET}_devilutionx_defconfig
-	BR2_JLEVEL=0 make toolchain libzip sdl sdl_mixer sdl_ttf
+	if [[ "$TARGET" != "rg350" ]]; then
```

Comment:
This shouldn't be done until performance issues are resolved.
This is tracked in <https://github.com/diasurgical/devilutionX/pull/542> so you can remove these changes from here

Reply:
Couldn't mention any performance issues on my RG350 so far, but i'll analyse it over time in different scenarios.